android/manifest.xsl

```diff
@@ -6,7 +6,7 @@
   <xsl:param name="onesignalAppID"></xsl:param>
 
   <xsl:template match="meta-data[@android:name='googleProjectNo']">
-    <meta-data android:name="googleProjectNo" android:value="\ {$googleProjectNo}"/>
```

Comment:
I think \ might be needed. did you verified this?

Reply:
When we tried the same with gameplay appID

java.lang.ClassCastException: java.lang.Float cannot be cast to java.lang.String

This error was coming.